nimare/meta/ibma.py

```diff
@@ -14,13 +14,33 @@
 from nipype.interfaces import fsl
 from nilearn.masking import unmask, apply_mask
 
+from ..utils import get_masker
 from .esma import fishers, stouffers, weighted_stouffers, rfx_glm
-from ..base import IBMAEstimator
+from ..base.estimators import Estimator
 from ..stats import p_to_z
 
 LGR = logging.getLogger(__name__)
 
 
+class IBMAEstimator(Estimator):
+    """Base class for image-based meta-analysis methods.
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
I'm a little confused about this. What are `*args` and `**kwargs` going to be here and why do we need to run the Estimator's init?

Reply:
Oh, that's vestigial. I briefly considered moving the masking stuff to the base class, but ultimately decided to leave it in `IBMAEstimator`, since it's not really relevant for CBMA.

Reply:
On a related note, we should make a decision about whether to enforce explicit inclusion in init signatures of all arguments that get passed onto superclasses/mixins. The tradeoff is usability vs. maintainability. Having to include `mask` in every `IBMAEstimator` would be a huge pain to maintain (have to update multiple docstrings each time any change is made, etc.), but would make it much clearer to users what functionality is available.

Personally I incline to keep it as-is for the time being, as it's easier to make args explicit later than to drop them in favor of `kwargs`. But am open to the alternative.